### ISSUE TITLE
Build error when publishing

### ISSUE THREAD

**Issue by ZsoltPath** [newrelic/nr1-github#44]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description
I've installed Nerdpack, set up profile and generated uuid.
But when I type `nr1 nerdpack:publish` it all seems fine in this window, but back in the server terminal I see the following error:
`ERROR in ./tmp/a355d06d-21ef-44eb-89e0-3ce37309af61-styles.1596193331921.scss`
See more below.
It seems to be running but when I connect to a GitHub repo I get `Oops, something went wrong` message without any further info. 

```
Found and loaded 1 nr1.json files on GitHub Integration (a355d06d-21ef-44eb-89e0-3ce37309af61) Nerdpack.

Nerdpack:
 ✔  GitHub Integration (a355d06d-21ef-44eb-89e0-3ce37309af61) nr1.json

Nerdlets:
 ✔  github nerdlets/github-about/nr1.json

🛠  Built artifact files for:
 ⁎  a355d06d-21ef-44eb-89e0-3ce37309af61--github built ✔

 ✔  Nerdpack built successfully!

Publishing Nerdpack GitHub Integration (a355d06d-21ef-44eb-89e0-3ce37309af61)
 ✔  Nerdpack published successfully!
 ✔  Deployed a355d06d-21ef-44eb-89e0-3ce37309af61 version 0.6.3 to the STABLE channel.
```

## Steps to Reproduce

```
nr1 nerdpack:clone -r https://github.com/newrelic/nr1-github.git
cd nr1-github/
nr1 nerdpack:serve
```
Other terminal
```
nr1 nerdpack:publish
```

## Expected Behaviour

Publish nerdpack without error

## Relevant Logs / Console output

```
🛠  Built artifact files for:
 ⁎  a355d06d-21ef-44eb-89e0-3ce37309af61--github built ✔

 ✔  Nerdpack built successfully!
Hash: ec7f7e306b1e1ff3a570
Version: webpack 4.42.1
Child
    Hash: ec7f7e306b1e1ff3a570
    Time: 24ms
    Built at: 07/31/2020 12:03:31 PM
                                                  Asset      Size                                        Chunks                   Chunk Names
        a355d06d-21ef-44eb-89e0-3ce37309af61--styles.js  5.32 KiB  a355d06d-21ef-44eb-89e0-3ce37309af61--styles  [emitted]        a355d06d-21ef-44eb-89e0-3ce37309af61--styles
    a355d06d-21ef-44eb-89e0-3ce37309af61--styles.js.map  3.72 KiB  a355d06d-21ef-44eb-89e0-3ce37309af61--styles  [emitted] [dev]  a355d06d-21ef-44eb-89e0-3ce37309af61--styles
    Entrypoint a355d06d-21ef-44eb-89e0-3ce37309af61--styles = a355d06d-21ef-44eb-89e0-3ce37309af61--styles.js a355d06d-21ef-44eb-89e0-3ce37309af61--styles.js.map
    [./tmp/a355d06d-21ef-44eb-89e0-3ce37309af61-styles.1596193331921.scss] 1.05 KiB {a355d06d-21ef-44eb-89e0-3ce37309af61--styles} [built] [failed] [1 error]

    ERROR in ./tmp/a355d06d-21ef-44eb-89e0-3ce37309af61-styles.1596193331921.scss
    Module build failed (from /usr/lib64/newrelic-cli/node_modules/mini-css-extract-plugin/dist/loader.js):
    ModuleBuildError: Module build failed (from /usr/lib64/newrelic-cli/node_modules/fast-sass-loader/lib/index.js):
    Error: ENOENT: no such file or directory, open '/home/zfejer/nr1-github/tmp/a355d06d-21ef-44eb-89e0-3ce37309af61-styles.1596193331921.scss'
        at runLoaders (/usr/lib64/newrelic-cli/node_modules/webpack/lib/NormalModule.js:316:20)
        at /usr/lib64/newrelic-cli/node_modules/loader-runner/lib/LoaderRunner.js:367:11
        at Array.<anonymous> (/usr/lib64/newrelic-cli/node_modules/loader-runner/lib/LoaderRunner.js:203:19)
        at Storage.finished (/usr/lib64/newrelic-cli/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:55:16)
        at ReadFileContext.provider (/usr/lib64/newrelic-cli/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:91:9)
        at ReadFileContext.callback (/usr/lib64/newrelic-cli/node_modules/graceful-fs/graceful-fs.js:115:16)
        at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:237:13)
    Child mini-css-extract-plugin ../../../usr/lib64/newrelic-cli/node_modules/css-loader/dist/cjs.js??ref--5-1!../../../usr/lib64/newrelic-cli/node_modules/postcss-loader/src/index.js??postcss!../../../usr/lib64/newrelic-cli/node_modules/fast-sass-loader/lib/index.js!tmp/a355d06d-21ef-44eb-89e0-3ce37309af61-styles.1596193331921.scss:
        Entrypoint mini-css-extract-plugin = *
        [../../../usr/lib64/newrelic-cli/node_modules/css-loader/dist/cjs.js?!../../../usr/lib64/newrelic-cli/node_modules/postcss-loader/src/index.js?!../../../usr/lib64/newrelic-cli/node_modules/fast-sass-loader/lib/index.js!./tmp/a355d06d-21ef-44eb-89e0-3ce37309af61-styles.1596193331921.scss] /usr/lib64/newrelic-cli/node_modules/css-loader/dist/cjs.js??ref--5-1!/usr/lib64/newrelic-cli/node_modules/postcss-loader/src??postcss!/usr/lib64/newrelic-cli/node_modules/fast-sass-loader/lib!./tmp/a355d06d-21ef-44eb-89e0-3ce37309af61-styles.1596193331921.scss 255 bytes {mini-css-extract-plugin} [built] [failed] [1 error]

        ERROR in ./tmp/a355d06d-21ef-44eb-89e0-3ce37309af61-styles.1596193331921.scss (/usr/lib64/newrelic-cli/node_modules/css-loader/dist/cjs.js??ref--5-1!/usr/lib64/newrelic-cli/node_modules/postcss-loader/src??postcss!/usr/lib64/newrelic-cli/node_modules/fast-sass-loader/lib!./tmp/a355d06d-21ef-44eb-89e0-3ce37309af61-styles.1596193331921.scss)
        Module build failed (from /usr/lib64/newrelic-cli/node_modules/fast-sass-loader/lib/index.js):
        Error: ENOENT: no such file or directory, open '/home/zfejer/nr1-github/tmp/a355d06d-21ef-44eb-89e0-3ce37309af61-styles.1596193331921.scss'
```
## Your Environment

* NR1 CLI version used: @datanerd/nr1/1.22.2 linux-x64 node-v10.16.3
* Browser name and version: Firefox v78.0.2
* Operating System and version: Windows 10, WSL2 Amazon linux 2

**Comment by danielgolden** [comment]
@ZsoltPath Sorry for the delay in response! I _am_ able to reproduce this, but I'm not sure why it's happening. I'm going to pull in some help from some colleagues who are better equipped to address this issue. Thank you for taking the time to log this issue.

**Comment by danielgolden** [comment]
**tldr**: Try publishing the nerdpack without serving it.

@ZsoltPath It looks like the issue here is that you're trying to publish the nerdpack _while_ also serving the nerdpack. `nr1 nerdpack:serve` is running a local webhost on files in `/dist`, when you execute `nr1 nerdpack:publish` it deletes everything in /dist in order to generate the artifact it needs to upload to publish it. So the errors about "file not found" are correct, b/c `nr1 nerdpack:publish` just deleted them all. These two process shouldn't happen at the same time, that's the source of your trouble.

Give this a go and let us know how it works out for you.
